indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -187,7 +187,9 @@ public TaskStatus call()
 
                               for (String propName : props.stringPropertyNames()) {
                                 for (String allowedPrefix : config.getAllowedPrefixes()) {
-                                  if (propName.startsWith(allowedPrefix)) {
+                                  // See https://github.com/druid-io/druid/issues/1841
+                                  if (propName.startsWith(allowedPrefix)
```

Comment:
ForkingTaskRunner is pretty much void of unit tests. I'm trying to add some in the Tiering extension, but unless really required I'm not interested in spending the time doing UTs for something I'm trying to replace.